package.json

```diff
@@ -13,12 +13,13 @@
     "build:clean": "rimraf dist && mkdirp dist",
     "build:quiet": "yarn build:clean && cross-env NODE_ENV=development webpack",
     "build": "yarn build:quiet --progress",
-    "build-prod:quiet": "yarn build:clean && yarn build-photon && cross-env NODE_ENV=production webpack",
-    "build-prod": "yarn build-prod:quiet --progress",
+    "build-prod:quiet": "yarn build:clean && yarn build-photon && cross-env NODE_ENV=production webpack && yarn build-sw",
+    "build-prod": "yarn build:clean && yarn build-photon && cross-env NODE_ENV=production webpack --progress && yarn build-sw",
```

Comment:
Is there a reason why we can't call `yarn build-sw` before the `webpack` command and keep build-prod as is? It looks like they are barely doing the same thing.